plasmapy/addons/__init__.py

```diff
@@ -0,0 +1,28 @@
+__all__ = []
+
+from pkg_resources import (iter_entry_points, resource_filename)
+
+
+def _parse_readme():
+    fname = resource_filename('plasmapy', 'addons/README.md')
+    with open(fname, encoding='utf-8') as f:
+        readme = f.read()
+
+    return readme
+
+
+__readme__ = _parse_readme()
+
+
+# import addon entry points
+for ep in iter_entry_points('plasmapy.addons'):
+    # __dict__[ep.name] = ep.load()
+    globals()[ep.name] = ep.load()
+    __all__.append(ep.name)
```

Comment:
A possibility would be to create and call a private function called something like `_import_addon_entry_points`.

Reply:
Not sure if that is necessary, since this isn’t functionality we’d use anywhere else and isn’t functionality we’d want exposed to users.

Reply:
I agree with @rocco8773, this is fine :)